diagnostic.md

```diff
@@ -6,21 +6,26 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    Tasks performed inside the Ember Application Router:
+    * tell the application which URL corresponds to which view
+
+    Tasks performed inside an Ember Route:
+    * parse the URL
+    * extract meaningful information (e.g. dynamic segments)
```

Comment:
More specifically to note is that the Route object specifies how to load model data through the model hook